cypress/integration/home.test.ts

```diff
@@ -0,0 +1,7 @@
+describe("The Home Page", () => {
+  it("successfully loads", () => {
+    cy.visit("/");
+
+    cy.findByRole("link", { name: /Sensoren ansehen/i }).should("exist");
```

Comment:
If possible we should use findByText in these cases. Since this is what the user actually sees. 
https://github.com/testing-library/cypress-testing-library/blob/310c394ed7c678fe4e6f6f582d258de62bf6a669/types/index.d.ts#L66

Reply:
Since these are just stub tests it's fine for now.

Reply:
I actually use `*ByRole` intentionally since this way we test against the accessibility tree and not just for users who access Stadtpuls visually. I'm aware that we are far from being accessibility-friendly, but it's good to approach this direction. Btw, this is the [recommended query priority by Testing Library itself](https://testing-library.com/docs/queries/about#priority).